R/get_ci.R

```diff
@@ -1,72 +1,156 @@
-#' Calculate a confidence interval
-#'
-#' @description Calculate a confidence interval given the estimate(s),
-#' the variance(s), the width of the interval, and the type of probability
-#' distribution to follow.
-#'
-#' @param dist A string to indicate the type of probability distribution
-#' to follow. Can be one of the following: `"normal"`, `"log normal"`, or `"poisson"`.
-#' This parameter is case insensitive.
-#' @param interval A scalar, between 0 and 1, indicating the width of the interval.
-#' For example, for a 95% confidence interval, use `interval = 0.95`.
-#' @param estimate A numeric vector containing the point estimates. When constructing
-#' a confidence interval using the Poisson distribution, `estimate` must be a (vector of)
-#' whole number(s). When the Poisson confidence interval is on a *rate*, `estimate` is the numerator.
-#' @param variance A numeric vector containing the variances. This parameter must
-#' be left empty when constructing a confidence interval for a Poisson distribution.
-#' @param denominator A numeric vector containing the denominator when constructing
-#' a Poisson confidence interval on a *rate*. Default value is
-#' `1` when `dist = "poisson"`. For other distributions, this parameter must be left empty.
-#'
-#' @details This low-level function doesn't need to know what the point estimates
-#' represent (e.g. rate, ratio, etc.). It also assumes that the estimates and variances
-#' are provided in the same order.
-#'
-#' `estimate` and `variance`, when provided, must have the same length. If `denominator` is
-#' provided, its length must be 1 or equal to the length of `estimate`. If the length of
-#' `denominator` is 1, R's [recycling rules](http://www.r-tutor.com/r-introduction/vector/vector-arithmetics) apply.
-#'
-#' The Poisson distribution is implemented using the chi-squared distribution to allow for
-#' the continuous extension of integer variable (`estimate`).
-#'
-#' @return A data frame with 2 columns `upper` and `lower` which contain the upper
-#' and lower bounds of a confidence interval respectively.
+#' Calculate a normal distribution confidence interval
```

Comment:
Is there a way to use roxygen2 to document the entire file / collection of functions? I think there is, but I don't recall the syntax off the top of my head. It would be helpful to remind the reader that these are *helper functions* and while they can be called by the user, they are intended to be called by the functions that calculate rates, risks, etc. (e.g. `get_ds_rt()`)  so many of the inputs for these functions are computed within these other functions.

Reply:
Using @seealso tag instead